public/stripejs-v3-mock.js

```diff
@@ -0,0 +1,50 @@
+// StripeJS fixture for using Stripe in feature specs. Mimics credit card form and Element objects.
```

Comment:
there must be a better place for this. ideally under spec/ otherwise, vendor?

Reply:
Agreed! I was planning on taking another pass at this after it was working, and now it's working :tada: